src/components/accordion/accordion.spec.js

```diff
@@ -0,0 +1,173 @@
+        styleNode.innerHTML = css;
+        document.head.appendChild(styleNode);
+    });
+
+    after(() => {
+        document.head.removeChild(styleNode);
+    });
+
+    it('The ui-accordion is an object', () => {
+        expect(UiAccordion).to.be.an('object');
+        expect(UiAccordion).to.be.not.empty;
+    });
+
+    it('It can be properly opened when the isPassive mode is disabled', () => {
+        const wrapper = shallowMount(UiAccordion, {
+            scopedSlots: {
+                head: '<h1 class="label">Head</h1>',
+                body: '<p>Body</p>',
+            },
+        });
+
+        const button = wrapper.find('button');
+        button.trigger('click');
+        expect(wrapper.vm.state.isOpen).to.be.ok;
+    });
+
+    it('It can be properly opened when the isPassive mode is enabled', () => {
+        const wrapper = shallowMount(UiAccordion, {
+            propsData: {
+                isPassive: true,
+                isOpen: false,
+            },
+            scopedSlots: {
+                head: '<h1 class="label">Head</h1>',
+                body: '<p>Body</p>',
+            },
+        });
+
+        wrapper.setProps({ isOpen: true });
+        expect(wrapper.vm.state.isOpen).to.be.ok;
+    });
+
+    it('Dom Events do not change the component state in the passive mode', () => {
+        const wrapper = shallowMount(UiAccordion, {
+            propsData: {
+                isPassive: true,
+                isOpen: false,
+            },
+            scopedSlots: {
+                head: '<h1 class="label">Head</h1>',
+                body: '<p>Body</p>',
+            },
+        });
+
+        const button = wrapper.find('button');
+        button.trigger('click');
+        expect(wrapper.vm.state.isOpen).to.be.not.ok;
+    });
+
+    it('The component change event will be dispatched', (done) => {
+        const wrapper = shallowMount(UiAccordion, {
+            propsData: {
+                isPassive: true,
+                isOpen: false,
+            },
+            scopedSlots: {
+                head: '<h1 class="label">Head</h1>',
+                body: '<p>Body</p>',
+            },
+            listeners: {
+                change() {
+                    expect(wrapper.vm.state.isOpen).to.be.ok;
+                    done();
+                },
+            },
+        });
+
+        wrapper.vm.$nextTick(() => {
+            wrapper.setProps({ isOpen: true });
+        });
+    });
+
+    it('The component changed event will be properly dispatched', (done) => {
+        const wrapper = shallowMount(UiAccordion, {
+            propsData: {
+                isPassive: true,
+                isOpen: false,
+            },
+            scopedSlots: {
+                head: '<h1 class="label">Head</h1>',
+                body: '<p>Body</p>',
+            },
+            listeners: {
+                changed(isOpen) {
+                    expect(wrapper.vm.state.isOpen).to.be.equal(isOpen);
```

Comment:
Can you change this to `true` or `false` according to the test you are running? The test should be as naive as possible, else we have a dependency on a variable defined in the test.

Reply:
I have made this test a bit more explicit